content/docs/command-reference/remote/modify.md

```diff
@@ -585,17 +585,32 @@ for a full guide on using Google Drive as DVC remote storage.
   $ dvc remote modify myremote gdrive_client_secret 'client-secret'
   ```
 
-- `gdrive_user_credentials_file` - path where DVC stores OAuth credentials to
-  access Google Drive data. `.dvc/tmp/gdrive-user-credentials.json` by default.
+- `gdrive_user_credentials_file` - global path where DVC stores OAuth
+  credentials to access Google Drive, by default
+  `$CACHE_HOME/pydrive2fs/{gdrive_client_id}/default.json` (see also
+  `profile` below), where the`CACHE_HOME` location per platform is:
+
+  | macOS              | Linux  (\*typical) | Windows                 |
+  | ------------------ | ------------------ | ----------------------- |
+  | `~/Library/Caches` | `~/.cache`         | `%CSIDL_LOCAL_APPDATA%` |
 
   ```dvc
   $ dvc remote modify myremote \
-        gdrive_user_credentials_file .dvc/tmp/mycredentials.json
+        gdrive_user_credentials_file path/to/mycredentials.json
   ```
 
 See [Authorization](/doc/user-guide/setup-google-drive-remote#authorization) for
 more details.
 
+- `profile` - name for the GDrive credentials cache directory (part of the
+  global `gdrive_user_credentials_file` path). By default, credentials are cached
```

Comment:
> "part of the default gdrive_user_credentials_file path" - where does it come from? why do you think it's part of that path?

@shcheklein so that's not the case? What I understood from this change is that it's `$CACHE_HOME/pydrive2fs/{gdrive_client_id}/default.json` by default, but if you set a `profile` then it becomes `$CACHE_HOME/pydrive2fs/{profile}/default.json`? Cc @efiop

Reply:
Yes, except small correction - `$CACHE_HOME/pydrive2fs/{profile}/default.json` should be `$CACHE_HOME/pydrive2fs/{client_id}/profile.json`

Anyways, the question is why specifically mentioning this parameter - `gdrive_user_credentials_file` ? `gdrive_user_credentials_file` redefined it completely as far as I understand (`profile` then is not taken into account). cc @efiop could you confirm?

Reply:
I confirm 🙂

Reply:
> why specifically mentioning this parameter - gdrive_user_credentials_file

Maybe I'm still confused. `profile` changes the default `gdrive_user_credentials_file`, right? Specifically the last part (file name). If so, there's a direct relationship.

Reply:
> why specifically mentioning this parameter - gdrive_user_credentials_file

Maybe I'm still confused. `profile` changes the default `gdrive_user_credentials_file`, right? Specifically the last part (file name). If so, there's a direct relationship.

Reply:
> gdrive_user_credentials_file redefined it completely

What happens if you set both `profile` and `gdrive_user_credentials_file`? Which one wins?

Reply:
`profile` is ignored in this case (if it's not it's a bug). See the comment here https://github.com/iterative/dvc.org/pull/3795#discussion_r932667184